CHANGELOG.md

```diff
@@ -10,7 +10,9 @@
 
 #### Enhancements
 
-* None.
+* Add `empty_collection_literal` rule to prefer using `isEmpty` to comparison to `[]` or `[:]`.  
+  [Colton Schlosser](https://github.com/cltnschlosser)
+  [2807](https://github.com/realm/SwiftLint/issues/2807)
```

Comment:
since you'll have to fix a conflict here, can you also add `#` before the issue number?

Reply:
can you also add that this is an opt-in rule?